src/services/git/IGitHubService.ts

```diff
@@ -5,9 +5,9 @@ export enum GitHubPullRequestState {
 }
 
 export enum GitHubGqlPullRequestState {
-  open = 'OPEN',
-  closed = 'CLOSED',
-  merged = 'MERGED',
+  open = 'is:open',
+  closed = 'is:closed',
+  all = '',
 }
```

Comment:
What about merged PRs? `is:closed is:merged` or just closed `is:closed is:unmerged`?